scenarioo-client/app/build/use-cases-overview/use-cases-overview.component.ts

```diff
@@ -182,7 +182,7 @@ export class UseCasesOverviewComponent {
 
     createBranchInformationTree(branch) {
         const branchInformationTree: any = {};
-        branchInformationTree.description = branch.description;
+        branchInformationTree.Description = branch.description;
```

Comment:
does this really need to be upper case?

Reply:
If it's not upper case here, is is written in lower case in the detail area:
![detailarea](https://user-images.githubusercontent.com/47637371/66287915-b7c0f680-e8d7-11e9-9133-884cd555278f.png)

Would it be better to set
`nodeLabel: key.charAt(0).toUpperCase() + key.slice(1),` 
on line 64 in treeDataCreator.pipe.ts?

Reply:
no no this should be done only in the view when outputting the keys and not in the data structure that is passed - so the details accordion itself should take care of it, no matter what keys are passed to it to always display the names of the keys in a human readable way (spaces and upper case first letter)

Reply:
please handle this inisde the component that renders these titles and not whewre the data is passed to it. We have other places where lowercase keys will be passed to accordions that need to be handled in general by the component in one place.

Reply:
@danielsuter: I looked at it with Rolf and we decided to keep it that way. Whoever writes node labels by hand is also responsible for whether this should be written in upper or lower case.